app/Services/Manager.php

```diff
@@ -495,4 +495,27 @@ public function addUserToModel($model, $id, $user)
 
         return true;
     }
+
+    /**
+     * Fires off an event to send an email a user.
+     *
+     * @param  Gladiator\Models\User     $user
+     * @param  Gladiator\Models\Contest  $contest
+     */
+    public function sendEmail($user, $contest, $params)
+    {
+        $message = Message::where($params)->first();
```

Comment:
should `$params = []` in the function def in case no params are passed? or maybe there's a check inside the method and returns out if no params passed? :shrug: